x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/nlp/SentimentAnalysisProcessor.java

```diff
@@ -56,7 +78,9 @@ InferenceResults processResult(PyTorchResult pyTorchResult) {
         }
 
         double[] normalizedScores = NlpHelpers.convertToProbabilitiesBySoftMax(pyTorchResult.getInferenceResult()[0]);
-        return new SentimentAnalysisResults(normalizedScores[1], normalizedScores[0]);
+        // the second score is usually the positive score so put that first
```

Comment:
This confused me. Now we have explicit labels for class 1 and class 2, shouldn't class 1 correspond to the first score (index 0) and class 2 to the second (index 1)?

Reply:
Yes that is part to the change 

`classLabels.get(1), normalizedScores[1]` is a pair and so is `classLabels.get(0), normalizedScores[0]`

Reply:
My confusion is that we pass the second score first in combination with the comment. Now we have explicit labels, shouldn't this be assumption free? (the assumption being that `the second score is usually the positive score`)

Reply:
We either flip the order here or in the `SentimentAnalysisResults` class. `SentimentAnalysisResults` has no concept of positive or negative sentiments all it knows is there is a class 1 score and class 1 label and a class 2 score and class 2 label. It makes more sense to me to have all the logic in one place and use `SentimentAnalysisResults` as a POJO.

The reason I am doing this is simply so that the positive result is first:
```
{
  "happy" : 0.9998486920335018,
  "sad" : 1.513079664982733E-4
}
```

rather than
```
{
  "sad" : 1.513079664982733E-4,
  "happy" : 0.9998486920335018  
}
```

I know ordering is not guaranteed in JSON but in effect the order is preserved and reads better in dev console. 

I updated the comment to say this ^